src/test/java/com/wikia/webdriver/common/remote/operations/GetSiteId.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.common.remote.operations;
+
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.remote.operations.http.NoAuthOperation;
+import lombok.Getter;
+import org.apache.http.client.methods.HttpGet;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.http.client.methods.HttpGet'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)